pkg/kubectl/cmd/util/helpers.go

```diff
@@ -57,7 +56,7 @@ type debugError interface {
 // souce is the filename or URL to the template file(*.json or *.yaml), or stdin to use to handle the resource.
 func AddSourceToErr(verb string, source string, err error) error {
 	if source != "" {
-		if statusError, ok := err.(*errors.StatusError); ok {
+		if statusError, ok := err.(errors.APIStatus); ok {
```

Comment:
Note that this is still inconsistent with line 108 of this file, but it was like that before.